app/client/src/store/actions.ts

```diff
@@ -61,4 +61,40 @@ export default {
       commit('setStatus', { task: 'network', data: 'submitted' });
     });
   },
+  async getStatus({ commit, state }: { commit: Commit, state: RootState }) {
+    await axios.post(
+      `${config.server_url}/status`,
+      { hash: state.projectHash },
+      {
+        headers: {
+          'Content-Type': 'application/json',
+        },
```

Comment:
To avoid repetition, you might want to pull out doing a post with json header, with base url,  into a helper method, or consider using an `apiService`-type class as we do in other projects e.g. https://github.com/mrc-ide/wodin/blob/main/app/static/src/app/apiService.ts

Could be in another ticket if you use a full apiService class.

Reply:
Have added a helper function for now, but will work on the api service class in the next ticket.